Templates/ScriptCanvasNode/Template/README.MD

```diff
@@ -0,0 +1,55 @@
+## How to use this template
+
+Use this template to produce a Script Canvas project that may hold Script Canvas nodes. 
+
+This guide will assume that you have created a gem called: `YourGem`
+
+```
+scripts\o3de create-from-template -tp C:\dev\o3de\Templates\ScriptCanvasNode -dp C:\dev\o3de\Gems\YourGem\Code\Source\MyScriptCanvasNodes
+```
+
+## Steps after using this template
+
+After running this template, there are still some manual steps needed in order to make it compile.
+
+### Step 1.
+
+Add the following into Gems/`YourGem`/Code/CMakeLists.txt
+
+```
+add_subdirectory(Source/${Name})
+```
+
+### Step 2.
+
+Locate the following configurations:
+
+```
+NAME YourGem.Editor GEM_MODULE
+NAME ScriptCanvasO3DCon.Private.Object STATIC
+NAME YourGem ${PAL_TRAIT_MONOLITHIC_DRIVEN_MODULE_TYPE}```
+```
```

Comment:
Another way to do this is to have the generated node CMakeLists.txt itself add itself as a dependency of the above targets.

The create-from-template command supports a `-r/--replace` option that allows specify a set of replacement pairs for names in a template.

That can be specified by the user to indicate which place "${...}" curly brace variables to replace.

At the end of the ScriptCanvasNode template CMakeLists.txt the following psuedocode can be added to allow users to specify the target build dependencies in the create-from-template-command.
```
foreach (dependent_target IN LISTS dependent_targets)
    target_link_libraries(${dependent_target} PRIVATE ${Name}.Static)
endforeach()
```

Also there is an extra set of three backticks at the end of line 30